code/modules/mob/living/status_procs.dm

```diff
@@ -150,6 +150,10 @@
 	if(!disabilities[disability])
 		return
 
+	if(!sources) // No defined source cures the disability entirely.
+		disabilities -= disability
+		return
+
```

Comment:
This should in theory already happen in line 164-165, does that not work?

Reply:
nope